### DISCUSSION TITLE
incomplete download, 400% complete???

### DISCUSSION THREAD

**Discussion by spawnreaper** [iterate-ch/cyberduck#14835] [Q&A]
needed to download some stuff from my dads server. It doesnt seem to finish the download, instead it just keeps going and going endlessly.
Before this happend I get some messages that the connection broke but it gave me the option to continue. All seemed fine after a couple of times I marked  the "always do this" box. 
And now I cant download anything. It keeps failing to complete to 100%, it just keeps going and never completes.
Anybody else have this problem?

**Comment by dkocher** [comment]
We are tracking this problem in #13374. Try to uncheck _Preferencers → Transfers → Downloads → Segmented downloads with multiple connections per file_.
